test.bash

```diff
@@ -30,21 +30,34 @@ cond_redirect() {
 
 # What test case should be run?
 if [ "$1" == "docker-full" ]; then
-    echo_process "Starting Docker based test..."
-    cond_redirect docker stop install-test || true
-    cond_redirect docker rm install-test || true
-    cond_redirect docker build --tag openhabian/openhabian-bats .
-    cond_redirect docker run -it openhabian/openhabian-bats bash -c 'bats -r -f "unit-." .'
-    cond_redirect docker run --name "install-test" --privileged -d openhabian/openhabian-bats
-    cond_redirect docker exec -it install-test bash -c "./build.bash local-test && mv ~/.profile ~/.bash_profile && /etc/rc.local"
-    cond_redirect docker exec -it install-test bash -c 'bats -r -f "installation-." .'
-    cond_redirect docker exec -it install-test bash -c 'bats -r -f "destructive-." .'
-    echo_process "Test complete, please review result in terminal. Access tested container by executing: \"docker exec -it install-test bash\""
-    exit 0
+  echo_process "Starting Docker based test..."
+  cond_redirect docker stop install-test || true
+  cond_redirect docker rm install-test || true
+  cond_redirect docker build --tag openhabian/openhabian-bats .
+  cond_redirect docker run -it openhabian/openhabian-bats bash -c 'bats -r -f "unit-." .'
+  cond_redirect docker run --name "install-test" --privileged -d openhabian/openhabian-bats
+  cond_redirect docker exec -it install-test bash -c "./build.bash local-test && mv ~/.profile ~/.bash_profile && /etc/rc.local"
+  cond_redirect docker exec -it install-test bash -c 'bats -r -f "installation-." .'
```

Comment:
can we parallelize this ?

Reply:
Good question, I don't actually know.

Reply:
Good question, I don't actually know.

Reply:
well if you left that to Travis .... fork 3x test.bash with arguments maybe ?

Reply:
What do you mean, I was just using it to setup Travis tests as it was simpler, I left the regular code for the user to run local tests. I also updated the local test code to default to amd64.